ftest.sh

```diff
@@ -205,7 +205,7 @@ export D_LOG_FILE=\"$TEST_TAG_DIR/daos.log\"
 mkdir -p ~/.config/avocado/
 cat <<EOF > ~/.config/avocado/avocado.conf
 [datadir.paths]
-logs_dir = $DAOS_BASE/src/tests/ftest/avocado/job-results
+logs_dir = $DAOS_BASE/install/lib/daos/TESTING/ftest/avocado/job-results
```

Comment:
Ditto.

Reply:
https://jira.hpdd.intel.com/browse/DAOS-3607
Will move CI test to DAOS RPM env.